recipes/ribocode/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
you can remove this file entirely as you have this command already in your meta.yaml file